src/app/shared/shared.module.ts

```diff
@@ -1,11 +1,17 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
```

Comment:
Your navigationcomponent is in the shared module and should also be declared and exported here, not in the app.module.